include/qfusion.hpp

```diff
@@ -0,0 +1,177 @@
+//////////////////////////////////////////////////////////////////////////////////////
+//
+// (C) Daniel Strano and the Qrack contributors 2017, 2018. All rights reserved.
+//
+// QUnit maintains explicit separability of qubits as an optimization on a QEngine.
```

Comment:
Probably need to update this header comment.

Reply:
Updated.